src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskAssignByIdControllerTest.java

```diff
@@ -222,6 +223,113 @@ void should_return_500_with_application_problem_response_when_assign_call_fails(
             ));
     }
 
+    @ParameterizedTest
+    @CsvSource(value = {
+        "IA, Asylum",
+        "WA, WaCaseType"
+    })
+    void should_return_500_with_application_problem_response_when_assign_call_fails_due_to_missing_assigneeId(
```

Comment:
small one. test name should be all snake case. assigneeId should be assignee_id

Reply:
actioned